src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/high-risk-data-breaches/View.tsx

```diff
@@ -0,0 +1,39 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// "use client";
+
+import { GuidedExperienceBreaches } from "../../../../../../../functions/server/getUserBreaches";
+import Link from "next/link";
+import { HighRiskBreachLayout } from "./HighRiskBreachLayout";
+
+type Props = {
+  breaches: GuidedExperienceBreaches;
+};
+export const View = (props: Props) => {
+  return (
+    <div>
+      <HighRiskBreachLayout
+        typeOfBreach="creditCard"
+        breachData={props.breaches}
+      />
+
+      <Link href="/redesign/user/dashboard/fix/high-risk-data-breaches/social-security-number">
+        SSN Breaches
+      </Link>
+      <br />
+      <Link href="/redesign/user/dashboard/fix/high-risk-data-breaches/credit-card-number">
+        Credit card
```

Comment:
Q: SHould this be "Credit Card" (uppercase Card) to match "Bank Account" below.
Also, why "SSN Breaches" but not "Credit Card Breaches" and "Bank Account Breaches" and "PIN Breaches" for consistency?